src/dmd/backend/dwarfdbginf.d

```diff
@@ -1105,7 +1113,17 @@ version (MARS) version (none)
     if (I64)
         debuginfo.address_size = 8;
 
-    debug_info.buf.write(&debuginfo, debuginfo.sizeof);
+    // Workaround https://issues.dlang.org/show_bug.cgi?id=16563
+    // Struct alignment is ignored due to 2.072 regression.
+    static if (debuginfo.alignof == 1)
```

Comment:
Actually, you can just check that the .sizeof is correct, that way the code will automatically upgrade when the build compiler is!